.changeset/seven-pots-wave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/dlc-btc-por-adapter': patch
```

Comment:
I wonder if this should be a major version upgrade since we introduce a breaking change as the old contract won't work anymore. On the other hand i don't think NOPs are using this EA as of now

Reply:
let's go with what you feel is correct, I'm OK either way. Good call out though 👍